### PR TITLE
Indicate macro exists as a bug workaround.

### PR DESCRIPTION
**Pull request by frewsxcv**
This change is [Reviewable](https://reviewable.io/reviews/servo/webrender/638)

### CODE REVIEW

File: webrender/res/prim_shared.glsl [hunk 1]
```diff
@@ -74,6 +74,9 @@ in int aLayerIndex;
 in int aElementIndex;
 in ivec2 aUserData;
 
+// get_fetch_uv is a macro to workaround a macOS Intel driver parsing bug.
```
Comment (kvark) [inline webrender/res/prim_shared.glsl]: I'd add `//TODO: convert back to a function once the driver issues are resolved, if ever`
Also, "workaround" here should be "work around" for a verb is expected.

Comment (frewsxcv) [inline webrender/res/prim_shared.glsl]: Done.

**Pull request merged**